client-cli/src/main/java/org/eclipse/dataspaceconnector/registration/cli/RegistrationServiceCli.java

```diff
@@ -14,27 +14,46 @@
 
 package org.eclipse.dataspaceconnector.registration.cli;
 
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import okhttp3.OkHttpClient;
+import org.eclipse.dataspaceconnector.iam.did.web.resolution.WebDidResolver;
 import org.eclipse.dataspaceconnector.registration.client.api.RegistryApi;
+import org.eclipse.dataspaceconnector.spi.monitor.ConsoleMonitor;
+import org.jetbrains.annotations.NotNull;
 import picocli.CommandLine;
 import picocli.CommandLine.Command;
 
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.concurrent.TimeUnit;
+
+import static java.lang.Boolean.parseBoolean;
+import static org.eclipse.dataspaceconnector.common.configuration.ConfigurationFunctions.propOrEnv;
+import static org.eclipse.dataspaceconnector.registration.cli.ClientUtils.createApiClient;
 
 @Command(name = "registration-service-cli", mixinStandardHelpOptions = true,
         description = "Client utility for MVD registration service.",
         subcommands = {
                 ParticipantsCommand.class
         })
 public class RegistrationServiceCli {
-    @CommandLine.Option(names = "-s", required = true, description = "Registration service URL", defaultValue = "http://localhost:8182/authority")
+
+    private static final ObjectMapper MAPPER = new ObjectMapper().disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
+    private static final String USE_HTTPS_SCHEME = "did.web.use.https";
+
+    @Deprecated
+    @CommandLine.Option(names = "-s", description = "Registration service URL. Deprecated. Use -d instead.", defaultValue = "http://localhost:8182/authority")
     String service;
 
-    @CommandLine.Option(names = "-d", required = true, description = "Client DID")
+    @CommandLine.Option(names = { "-d", "--dataspace-did" }, required = false, description = "Dataspace Authority DID.", defaultValue = "")
```

Comment:
```suggestion
    @CommandLine.Option(names = { "-d", "--dataspace-did" }, description = "Dataspace Authority DID.", defaultValue = "")
```

Reply:
I left this one here because it will later change to required=true after removal of option '-s'

Reply:
You should add an annotation to ignore warnings if this is really warranted (though in that case I don't really see the reason - why can't it be changed to required=true after it's been removed?) so that our IDE doesn't get polluted and we don't need to mentally sort warnings that should be ignored vs those that shouldn't

Reply:
Of course it can be added later, I just decided to keep it to make a difference between this one ald the -s one that the `required` is there for a reason. In the next PR it's going to be changed to `true`, but it's not worth a big discussion I think, let me remove it